modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/utils/api.ts

```diff
@@ -145,15 +145,15 @@ async function deleteItem(url: string) {
 	}
 }
 
-export async function addPickListItem({
-	id,
+export async function postListTypeEntry({
```

Comment:
@igor-franca Also sort this function in the file.

Reply:
It has already been done in the sort commit